pkg/local-storage/member/node/volume_task_worker.go

```diff
@@ -91,28 +96,68 @@ func (m *manager) processVolumeReplicaTaskAssignment(volName string) error {
 }
 
 func (m *manager) createVolumeReplica(vol *apisv1alpha1.LocalVolume) error {
+	m.lock.Lock()
+	defer m.lock.Unlock()
 	if replicaName, exists := m.replicaRecords[vol.Name]; exists {
 		m.logger.WithField("replica", replicaName).Debug("LocalVolumeReplica exists, ignore the creation")
 		return nil
 	}
-	replica := &apisv1alpha1.LocalVolumeReplica{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: fmt.Sprintf("%s-%s", vol.Name, utilrand.String(6)),
-		},
-		Spec: apisv1alpha1.LocalVolumeReplicaSpec{
-			VolumeName:            vol.Name,
-			PoolName:              vol.Spec.PoolName,
-			RequiredCapacityBytes: vol.Spec.RequiredCapacityBytes,
-			NodeName:              m.name,
-		},
+
+	var srcPvcName, srcVolumeName string
+	if strings.Contains(vol.Spec.PersistentVolumeClaimName, migratePvcSuffix) {
+		splitRes := strings.Split(vol.Spec.PersistentVolumeClaimName, migratePvcSuffix)
+		srcPvcName = splitRes[0]
+
+		srcPvc := &corev1.PersistentVolumeClaim{}
+		if err := m.apiClient.Get(context.TODO(), types.NamespacedName{Name: srcPvcName, Namespace: vol.Spec.PersistentVolumeClaimNamespace}, srcPvc); err != nil {
+			if !errors.IsNotFound(err) {
+				m.logger.WithError(err).Error("createVolumeReplica Failed to query srcPvc")
+			} else {
+				m.logger.Info("createVolumeReplica Not found the srcPvc")
+			}
+			return err
+		}
+
+		srcVolumeName = srcPvc.Spec.VolumeName
+	}
+
+	var replica = &apisv1alpha1.LocalVolumeReplica{}
+
+	if srcVolumeName == "" {
+		replica = &apisv1alpha1.LocalVolumeReplica{
```

Comment:
The `replica` variable has allocated at line:124, why do you allocate once more?